Cargo.toml

```diff
@@ -24,6 +24,15 @@ include = [
 license = "MIT OR Apache-2.0"
 readme = "README.md"
 
+[workspace.dependencies]
+clap = { path = ".", version = "3.1.10", default-features = false, features = ["std"] }
+clap_derive = { path = "./clap_derive", version = "=3.1.18" }
+clap_lex = { path = "./clap_lex", version = "0.2.0" }
+os_str_bytes = { version = "6.0", default-features = false, features = ["raw_os_str"] }
```

Comment:
FYI this is needed to be shared because its a public dependency of `clap_lex` though `clap_lex` re-exports parts of it, so maybe we could pull this from `clap` depending on it?

Reply:
What do you mean?